README.md

```diff
@@ -15,3 +15,10 @@ This works exactly as through you have called: `OsRng.gen::<u32>()` at compile t
 So for details of the random number generation, see the `rand` crates documentation.
 
 The following types are supported: u8, i8, u16, i16, u32, i32, u64, i64, u128, i128, usize, and isize. 
+
+# Deterministic builds
+
+Sometimes it is an advantage for build systems to be deterministic. To support this `const-random` reads the environmental
+variable `CONST_RANDOM_SEED`. If this variable is set, it will be used as the seed for the random number generation.
+Setting the same seed on a build of the same code should result in identical output.
```

Comment:
I really think it should be deterministic by default. It is very easy to accidentally depend on a crate that would cause your whole crate graph to not be reproducible. Non-determinism also works very bad with caching.

Reply:
What could be a default source of randomness?